pkgs/development/libraries/amtk/default.nix

```diff
@@ -0,0 +1,36 @@
+{ stdenv, autoconf, autoconf-archive, automake, intltool, gnome2, pkgconfig
+, libtool, gobjectIntrospection, gnome3 }:
+let
+  version = "4.99.1";
+in stdenv.mkDerivation {
+  name = "amtk-${version}";
+  src = fetchTarball {
+    url = "https://gitlab.gnome.org/GNOME/amtk/-/archive/${version}/amtk-${version}.tar.bz2";
```

Comment:
You should probably use `mirror://gnome/sources/amtk/${gnome3.versionBranch version}/${name}.tar.xz`, since it will be served by geographically distributed mirrors.

Reply:
Also sorry about misinformation, we now have `stdenv.lib.versions.majorMinor` which should be preferred.